client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -140,16 +156,35 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 	const connectComponent = connect(
 		state => {
 			const siteId = getSelectedSiteId( state );
-			const isRequestingSettings = isRequestingSiteSettings( state, siteId );
-			const isSavingSettings = isSavingSiteSettings( state, siteId );
-			const isSaveRequestSuccessful = isSiteSettingsSaveSuccessful( state, siteId );
-			const settings = getSiteSettings( state, siteId );
+			let isSavingSettings = isSavingSiteSettings( state, siteId );
+			let isSaveRequestSuccessful = isSiteSettingsSaveSuccessful( state, siteId );
+			let settings = getSiteSettings( state, siteId );
+			let isRequestingSettings = isRequestingSiteSettings( state, siteId ) && ! settings;
+			const siteSettingsSaveError = getSiteSettingsSaveError( state, siteId );
+			const settingsFields = {
+				site: keys( settings ),
+			};
+
+			const isJetpack = isJetpackSite( state, siteId );
+			const jetpackSettingsUISupported = isJetpack && siteSupportsJetpackSettingsUI( state, siteId );
```

Comment:
Even acronyms should follow camel-casing recommendations:

https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions

i.e. `siteSupportsJetpackSettingsUi`